tests/checks/common.py

```diff
@@ -33,6 +33,10 @@
 AUTO_CONFD_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', '..', 'conf.d', 'auto_conf')
 
 def copy_checks():
+    if not os.path.exists(CHECKSD_PATH):
```

Comment:
I saw the convo the other day regarding this. We should maybe add a card or a repo issue or something to not forget tweaking this and avoid having to copy/remove the checks.